README.md

```diff
@@ -137,6 +137,70 @@ export default withOnyx({
 
 It is preferable to use the HOC over `Onyx.connect()` in React code as `withOnyx()` will delay the rendering of the wrapped component until all keys have been accessed and made available.
 
+## Collections
+
+Collections allow keys with similar value types to be subscribed together by subscribing to the collection key. To define one, it must be included in the `ONYXKEYS.COLLECTION` object and it must be suffixed with an underscore. Member keys should use a unique identifier or index after the collection key prefix (e.g. `chat_42`).
```

Comment:
NAB but should we use an existing key as an example? That would make it easier for a contributor to search for examples in the code base and see how collections are used.

```suggestion
Collections allow keys with similar value types to be subscribed together by subscribing to the collection key. To define one, it must be included in the `ONYXKEYS.COLLECTION` object and it must be suffixed with an underscore. Member keys should use a unique identifier or index after the collection key prefix (e.g. `report_42`).
```

Reply:
Ah so, my intention here was to try to keep this generic so that people could imagine using it for whatever their use case and not get thrown off by "expensify" language. But I think the point gets across well enough here either way though so I'll update it.

Reply:
Makes sense! Thanks for the changes